spec/intl.html

```diff
@@ -304,7 +304,11 @@ <h1>PartitionDateTimePattern ( _dateTimeFormat_, <del>_x_</del><ins>_date_</ins>
           1. <ins>If _date_ has an [[InitializedTemporalTime]] internal slot, then</ins>
             1. <ins>Let _pattern_ be _dateTimeFormat_.[[TemporalTimePattern]].</ins>
             1. <ins>Let _tm_ be { [[hour]]: _date_.[[Hour]], [[minute]]: _date_.[[Minute]], [[second]]: _date_.[[Second]] }.</ins>
-          1. <ins>If _date_ has an [[InitializedTemporalDateTime]] or an [[InitializedTemporalInstant]] internal slot, then</ins>
+          1. <ins>If _date_ has an [[InitializedTemporalInstant]] internal slot, then</ins>
+            1. <ins>Let _timeZone_ be ? TimeZoneFrom(_dateTimeFormat_.[[TimeZone]]).</ins>
+            1. <ins>Let _calendar_ be ? CalendarFrom(_dateTimeFormat_.[[Calendar]]).</ins>
+            1. <ins>Set _date_ to ? GetTemporalDateTimeFor(_timeZone_, _instant_, _calendar_).</ins>
+          1. <ins>If _date_ has an [[InitializedTemporalDateTime]] internal slot, then</ins>
             1. <ins>Let _calendar_ be ? CalendarToString(_x_).</ins>
```

Comment:
Followup: all of these are wrong. Should be `CalendarToString(_date_.[[Calendar]])`.